cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
No I mean `_properties_form.properties_form` makes no sense. `_properties_form` obviously can't be a properties form if it contains a properties form. So we might as well name it properly.